### ISSUE TITLE
ELMS, read to me

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#1645]
Using the speech synth API native in browsers we should be able to basically turn courses into book-on-tape page turners (obviously with tinkling stars to indicate to turn the page...). We could take the body of the article, and feed it in 1 word at a time. This would allow us to do an underlining while it reads to suggest what word you are on.
- [x] read on voice command
- [x] button on interface for read to me mode
- [x] ability to stop speaking w/ "stop" word recognized
- [ ] UI widget to pause / go into read to me mode (turns the page, saying it will turn the page, etc)
- [ ] scroll to the word in view / several words ahead of where it is
- [ ] cursor that's underlining the word it's reading OR, highlight all words its going to read and focus on the one its reading as it goes by

**Comment by btopro** [comment]
currently a checkbox, needs to be a button / little UI widget of its own
